examples/scenes/src/download.rs

```diff
@@ -68,12 +69,13 @@ impl Download {
                             builtin.license,
                             builtin.info
                         );
+                        total_bytes += builtin.expected_size;
                     }
 
                     // For rustfmt, split prompt into its own line
-                    const PROMPT: &str =
-                "Would you like to download a set of default lottie files, as explained above?";
-                    accepted = Confirm::new(PROMPT).with_default(false).prompt()?;
+                    let prompt =
+                        format!("Would you like to download a set of default lottie files, as explained above? ({})", byte_unit::Byte::from_bytes(total_bytes.into()).get_appropriate_unit(false));
```

Comment:
This line is running into https://github.com/rust-lang/rustfmt/issues/3135 -
```sh
fsh ❯ node
Welcome to Node.js v20.11.1.
Type ".help" for more information.
> `format!("Would you like to download a set of default lottie files, as explained above? ({})", byte_unit::Byte::from_bytes(total_bytes.into()).get_appropriate_unit(false));`.length
171
> 
```

We need to format it properly to be within the normal line length limit.
We might need to use a similar trick as in linebender/vello#499 here, that is moving it into its own top-level function

Reply:
I don't see fmt complaining in the CI. Why not just leave it as is?

Reply:
Because it's not following the Rust [Style Guide](https://doc.rust-lang.org/nightly/style-guide/index.html#indentation-and-line-width).

Just because rustfmt breaks isn't an especially compelling reason to not do so. It also stands out significantly as an unreasonably long line

Reply:
fixed